src/Sarif.Driver/Sdk/SkimmerBase.cs

```diff
@@ -23,7 +23,9 @@ public SkimmerBase()
 
         abstract protected ResourceManager ResourceManager { get; }
 
-        abstract protected IEnumerable<string> TemplateIds { get; }
+        abstract protected IEnumerable<string> TemplateResourceIds { get; }
```

Comment:
>TemplateResourceIds [](start = 47, length = 19)

Maybe 'TemplateResourceNames' is even better?

Reply:
@michaelcfanning  You are right. In the API[ `ResourceManager.GetString(string)`](https://msdn.microsoft.com/en-us/library/d17ax2xk(v=vs.110).aspx), the name of the parameter is indeed `name` instead of `id`. Fixed.